ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/worldview/BonsaiWorldStateUpdateAccumulator.java

```diff
@@ -77,18 +79,20 @@ public class BonsaiWorldStateUpdateAccumulator
   public BonsaiWorldStateUpdateAccumulator(
       final BonsaiWorldView world,
       final Consumer<BonsaiValue<BonsaiAccount>> accountPreloader,
-      final Consumer<StorageSlotKey> storagePreloader) {
-    super(world);
+      final Consumer<StorageSlotKey> storagePreloader,
+      final EvmConfiguration evmConfiguration) {
+    super(world, evmConfiguration);
```

Comment:
Journaled updater only works post-merge right now.   My local node runs journaled updater eithout any flaws for about a month now.

That's where my advocacy for [EIP-7523](https://eips.ethereum.org/EIPS/eip-7523) got it's seed.  There are issues relating to empty account touches and reverts that I could spend 20 hours fixing, or just accept that journaled world state doesn't matter for ancient blocks.  However until that EIP and stuff like EIP-4444 gain traction and adoption I don't think it should be the default.

Reply:
> rather than plumbing evmConfiguration everywhere in order to get the updater config, could we just overload the AbstractWorldUpdater constructor without config and pass the default config to the other constructor?

This is a common DI by hand issue.  I need to set the EVM configuration in the constructor for the concrete class, even though only the abstract class is the one that uses it.